src/components/calcite-panel/calcite-panel.scss

```diff
@@ -131,11 +127,11 @@
 }
 
 .header-actions--end {
-  @apply ml-auto;
+  margin-inline-start: auto;
 }
 
 .menu-container:only-child {
-  @apply ml-auto;
+  margin-inline-start: auto;
```

Comment:
Nitpick: this could be merged into the above selector.

Reply:
Tailwind doesn't provide an equivalent selector for `margin-inline-start` including many if not all of the other CSS logical properties.